core/src/mindustry/content/UnitTypes.java

```diff
@@ -899,7 +904,8 @@ public void load(){
             faceTarget = false;
             armor = 4f;
 
-            weapons.add(new Weapon(){{
+            weapons.add(
+            new Weapon(){{
```

Comment:
and also here

Reply:
All the other weapons are like this, so I made these 3 like the others; make formatting consistent.

Yes I did just make the same comment to 3 comments.